LayoutTests/fast/canvas/webgl/debug-messages-to-console.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="/js-test-resources/js-test.js"></script>
```

Comment:
I don't think this file exists, you should see an error in Web Inspector.

instead, you should 
use the .../../resources/js-test.js 

use
`jsTestIsAsync = true`

This will make the test wait for function call jsTestFinished()